src/controllers/reporting.controller.ts

```diff
@@ -18,7 +18,23 @@ export const getZipAlertList = asyncHandler(
         req: Request<{}, {}, {}, {}>,
         res: Response,
         next: NextFunction
-    ) => {}
+    ) => {
+        //var arr: Number[] = []
+        var { zipList } = require('../config/queue.ts');
+        if(zipList != null){
+            if(zipList.length > 0){
```

Comment:
I think regardless of size, we should still send `zipList` as the response. If it's empty, it will just appear as `{ ziplist: []}` which should be fine.